src/translator/quality_estimator.h

```diff
@@ -0,0 +1,31 @@
+#ifndef SRC_BERGAMOT_QUALITY_ESTIMATOR_H_
+#define SRC_BERGAMOT_QUALITY_ESTIMATOR_H_
+
+#include <iostream>
+#include <fstream>
+
+#define ONNX_API
+#include "response.h"
+#include "quality_estimator.h"
+#include "3rd_party/onnx/onnx/onnx/onnx.pb.h"
+
+namespace marian {
+namespace bergamot {
+
+class QualityEstimator {
+  public:
+    onnx::ModelProto model;
+    Response response_;
+    QualityEstimator(Response &response, std::string onnx_path) {
```

Comment:
Your model (be it a neural network graph, logistic regression weights) will be loaded and kept in memory longer than one `Response`. The ideal plan is to instantiate `QualityEstimator` in `Service` and a reference to it given to `ResponseBuilder`. 

On each call to update `Response`, `ResponseBuilder` will call the same instance.